superset/examples/configs/charts/Filter_Segments.yaml

```diff
@@ -23,32 +23,37 @@ params:
   filter_configs:
   - asc: true
     clearable: true
-    column: Ethnic Minority
+    column: ethnic_minority
     key: -xNBqpfQo
+    label: Ethnic Minority
     multiple: true
     searchAllOptions: false
   - asc: true
     clearable: true
     column: gender
     key: 19VeBGTKf
+    label: Gender
     multiple: true
     searchAllOptions: false
   - asc: true
     clearable: true
-    column: Developer Type
+    column: developer_type
     key: OWTb4s69T
+    label: Developer Type
     multiple: true
     searchAllOptions: false
   - asc: true
     clearable: true
     column: lang_at_home
     key: Fn-YClyhb
+    label: Language at Home
     multiple: true
     searchAllOptions: false
   - asc: true
     clearable: true
     column: country_live
     key: 2fNskRCLJ
+    label: Country live
```

Comment:
small nit, do you want to capitalize Live?

Reply:
Good point, but TBH we might need to come up with something better. @srinify what should be the label here? "Country currently living on"?